Source/ACE/Network/Enum/Usable.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace ACE.Network.Enum
+{
+    public enum Usable : int
+    {
+        Undef       = 0x00,
+        No          = 0x01,
+        Self        = 0x02,
+        Wielded     = 0x04,
+        Contained   = 0x08,
+        Viewed      = 0x10,
+        Remote      = 0x20,
+        NeverWalk   = 0x40,
+        ObjSelf     = 0x80,
+
+        ContainedViewed                 = 24,
+        ContainedViewedRemote           = 56,
+        ContainedViewedRemoteNeverWalk  = 120,
+
+        ViewedRemote            = 48,
+        ViewedRemoteNeverWalk   = 112,
+
+        RemoteNeverWalk         = 96,
+
+        SourceWieldedTargetWielded          = 262148,
```

Comment:
I'm not sure how to fully resolve this one...

This is the flag pulled from the client... everything below RemoteNeverWalk doesn't seem to resolve in my testing without specifying as exported.

```
0x00015c3d : Length = 1690, Leaf = 0x1203 LF_FIELDLIST
	list[0] = LF_ENUMERATE, public, value = 0, name = 'USEABLE_UNDEF'
	list[1] = LF_ENUMERATE, public, value = 1, name = 'USEABLE_NO'
	list[2] = LF_ENUMERATE, public, value = 2, name = 'USEABLE_SELF'
	list[3] = LF_ENUMERATE, public, value = 4, name = 'USEABLE_WIELDED'
	list[4] = LF_ENUMERATE, public, value = 8, name = 'USEABLE_CONTAINED'
	list[5] = LF_ENUMERATE, public, value = 16, name = 'USEABLE_VIEWED'
	list[6] = LF_ENUMERATE, public, value = 32, name = 'USEABLE_REMOTE'
	list[7] = LF_ENUMERATE, public, value = 64, name = 'USEABLE_NEVER_WALK'
	list[8] = LF_ENUMERATE, public, value = 128, name = 'USEABLE_OBJSELF'
	list[9] = LF_ENUMERATE, public, value = 24, name = 'USEABLE_CONTAINED_VIEWED'
	list[10] = LF_ENUMERATE, public, value = 56, name = 'USEABLE_CONTAINED_VIEWED_REMOTE'
	list[11] = LF_ENUMERATE, public, value = 120, name = 'USEABLE_CONTAINED_VIEWED_REMOTE_NEVER_WALK'
	list[12] = LF_ENUMERATE, public, value = 48, name = 'USEABLE_VIEWED_REMOTE'
	list[13] = LF_ENUMERATE, public, value = 112, name = 'USEABLE_VIEWED_REMOTE_NEVER_WALK'
	list[14] = LF_ENUMERATE, public, value = 96, name = 'USEABLE_REMOTE_NEVER_WALK'
	list[15] = LF_ENUMERATE, public, value = (LF_ULONG) 262148, name = 'USEABLE_SOURCE_WIELDED_TARGET_WIELDED'
	list[16] = LF_ENUMERATE, public, value = (LF_ULONG) 524292, name = 'USEABLE_SOURCE_WIELDED_TARGET_CONTAINED'
	list[17] = LF_ENUMERATE, public, value = (LF_ULONG) 1048580, name = 'USEABLE_SOURCE_WIELDED_TARGET_VIEWED'
	list[18] = LF_ENUMERATE, public, value = (LF_ULONG) 2097156, name = 'USEABLE_SOURCE_WIELDED_TARGET_REMOTE'
	list[19] = LF_ENUMERATE, public, value = (LF_ULONG) 6291460, name = 'USEABLE_SOURCE_WIELDED_TARGET_REMOTE_NEVER_WALK'
	list[20] = LF_ENUMERATE, public, value = (LF_ULONG) 262152, name = 'USEABLE_SOURCE_CONTAINED_TARGET_WIELDED'
	list[21] = LF_ENUMERATE, public, value = (LF_ULONG) 524296, name = 'USEABLE_SOURCE_CONTAINED_TARGET_CONTAINED'
	list[22] = LF_ENUMERATE, public, value = (LF_ULONG) 8912904, name = 'USEABLE_SOURCE_CONTAINED_TARGET_OBJSELF_OR_CONTAINED'
	list[23] = LF_ENUMERATE, public, value = (LF_ULONG) 655368, name = 'USEABLE_SOURCE_CONTAINED_TARGET_SELF_OR_CONTAINED'
	list[24] = LF_ENUMERATE, public, value = (LF_ULONG) 1048584, name = 'USEABLE_SOURCE_CONTAINED_TARGET_VIEWED'
	list[25] = LF_ENUMERATE, public, value = (LF_ULONG) 2097160, name = 'USEABLE_SOURCE_CONTAINED_TARGET_REMOTE'
	list[26] = LF_ENUMERATE, public, value = (LF_ULONG) 6291464, name = 'USEABLE_SOURCE_CONTAINED_TARGET_REMOTE_NEVER_WALK'
	list[27] = LF_ENUMERATE, public, value = (LF_ULONG) 2228232, name = 'USEABLE_SOURCE_CONTAINED_TARGET_REMOTE_OR_SELF'
	list[28] = LF_ENUMERATE, public, value = (LF_ULONG) 262160, name = 'USEABLE_SOURCE_VIEWED_TARGET_WIELDED'
	list[29] = LF_ENUMERATE, public, value = (LF_ULONG) 524304, name = 'USEABLE_SOURCE_VIEWED_TARGET_CONTAINED'
	list[30] = LF_ENUMERATE, public, value = (LF_ULONG) 1048592, name = 'USEABLE_SOURCE_VIEWED_TARGET_VIEWED'
	list[31] = LF_ENUMERATE, public, value = (LF_ULONG) 2097168, name = 'USEABLE_SOURCE_VIEWED_TARGET_REMOTE'
	list[32] = LF_ENUMERATE, public, value = (LF_ULONG) 262176, name = 'USEABLE_SOURCE_REMOTE_TARGET_WIELDED'
	list[33] = LF_ENUMERATE, public, value = (LF_ULONG) 524320, name = 'USEABLE_SOURCE_REMOTE_TARGET_CONTAINED'
	list[34] = LF_ENUMERATE, public, value = (LF_ULONG) 1048608, name = 'USEABLE_SOURCE_REMOTE_TARGET_VIEWED'
	list[35] = LF_ENUMERATE, public, value = (LF_ULONG) 2097184, name = 'USEABLE_SOURCE_REMOTE_TARGET_REMOTE'
	list[36] = LF_ENUMERATE, public, value = (LF_ULONG) 6291488, name = 'USEABLE_SOURCE_REMOTE_TARGET_REMOTE_NEVER_WALK'
	list[37] = LF_ENUMERATE, public, value = (LF_USHORT) 65535, name = 'USEABLE_SOURCE_MASK'
	list[38] = LF_ENUMERATE, public, value = (LF_LONG) -65536, name = 'USEABLE_TARGET_MASK'
	list[39] = LF_ENUMERATE, public, value = (LF_ULONG) 2147483647, name = 'FORCE_ITEM_USEABLE_32_BIT'
```